canbench-bin/Cargo.toml

```diff
@@ -1,33 +1,33 @@
 [package]
 name = "canbench"
 # NOTE: whenever this version is updated, canbench-rs below should also be updated.
```

Comment:
@berestovskyy 

Heads-up: it looks like an editor plugin auto-sorted `Cargo.toml`, moving comments away from the fields they refer to. For example, the version comment is now detached. Not critical here, but could be confusing for other fields.